vault/resource_consul_secret_backend_test.go

```diff
@@ -173,6 +173,100 @@ func TestConsulSecretBackend_Bootstrap(t *testing.T) {
 	})
 }
 
+func TestConsulSecretBackend_remount(t *testing.T) {
+	path := acctest.RandomWithPrefix("tf-test-consul")
+	updatedPath := acctest.RandomWithPrefix("tf-test-consul-updated")
+	token := "026a0c16-87cd-4c2d-b3f3-fb539f592b7e"
+
+	resourceName := "vault_consul_secret_backend.test"
+
+	var oldUUID string
```

Comment:
I prefer to use a struct for this type of thing, and then pass the an instance ref around. This is okay for this case however.

Reply:
Updated to use a `testMountStore` object. Thanks!